app/gui2/src/util/shortcuts.ts

```diff
@@ -391,6 +391,9 @@ export function defineKeybinds<
     >,
   ): (event: Event_, stopAndPrevent?: boolean) => boolean {
     return (event, stopAndPrevent = true) => {
+      // Do not handle repeated keyboard events (held down key).
+      if (event instanceof KeyboardEvent && event.repeat) return false
```

Comment:
I think returning `false` unconditionally in this case risks bugs: If the first event was handled by a shortcut and stopped, but the repeat events are not stopped, they could be handled by something else--which would probably be surprising.